getting-started/consume-services.hbs.md

```diff
@@ -1,38 +1,27 @@
-
-- Set up a service.
-- Created a service instance.
-- Claimed the service instance.
-
-For more information about these prerequisites, see [Set up services for consumption by developers](set-up-services.md). Also, for important background, see [Consume services on Tanzu Application Platform](about-consuming-services.md).
+This tutorial walks the application developer through deploying two application workloads and configuring them to communicate via a service instance. It uses RabbitMQ as an example, but the process is the same regardless of the service you want to set up. You will use the `tanzu service` CLI plug-in and will learn about classes, claims and bindings.
 
 ## <a id="you-will"></a>What you will do
 
-- Inspect the claim created for the service instance by the application operator.
-- Bind the application workload to the claim so the workload utilizes the service instance.
+- Discover the range of services available to you
+- Create a claim for an instance of one of the services
+- Create two application workloads and bind them to the claim so that the workloads utilize the service instance
 
 ## <a id="overview"></a>Overview
 
-The following diagram depicts a summary of what this walkthrough covers, including the work of the service and application operators described in [Set up services for consumption by developers](set-up-services.md).
+The following diagram depicts a summary of what this tutorial covers.
 
-![Diagram shows the default namespace and service instances namespace. The default namespace has two application workloads, each connected to a service binding. The service bindings connect to the service instance in the service instances namespace through a claim.](../images/getting-started-stk-1.png)
+![Diagram shows the default namespace and a glimpse at what is happening behind the scenes. The default namespace has two application workloads, each connected to a service binding. The service bindings refer to a single claim, which refers to a service instance.](../images/getting-started-stk-1.png)
 
 Bear the following observations in mind as you work through this guide:
 
-1. There is a clear separation of concerns across the various user roles.
-
-    * Application developers set the life cycle of workloads.
-    * Application operators set the life cycle of claims.
-    * Service operators set the life cycle of service instances.
-    * The life cycle of service bindings is implicitly tied to the life cycle of workloads.
-
-2. ProvisionedService is the contract allowing credentials and connectivity information to flow from the service instance to the class claim, to the resource claim, to the service binding, and ultimately to the application workload. For more information, see [ProvisionedService](https://github.com/servicebinding/spec#provisioned-service) on GitHub.
+1. There are a set of four classes preinstalled on the cluster.
```

Comment:
**thought:** Unless they have been explicitly disabled by the Service/Platform Operator.

Reply:
I had the same thought, but I think given this is the "getting started" section of the docs it's probably ok to assume that they haven't been disabled.